Assets/Scripts/Yarn/LineView.cs

```diff
@@ -297,6 +297,12 @@ private void Reset()
             canvasGroup = GetComponentInParent<CanvasGroup>();
         }
 
+        private void Update()
+        {
+            // if hitting spacebar during update, trigger dialogue advancement
+            if (Input.GetButtonDown("Jump")) UserRequestedViewAdvancement();
+        }
+
```

Comment:
@jentaruno i just added this too, it allows spacebar to basically trigger a continue button press

Reply:
BOOTS